meeting-minutes.php

```diff
@@ -10,6 +10,10 @@
 
 
     <div class="container-fluid">
+        <div>
+        <p></p>Starting in July 2019, OBF Board public meeting minutes can be found in
```

Comment:
There's an extra ``</p>`` at the start, but this still seems to have had no effect.

Perhaps it should be within the <article> section...

Reply:
See #90. Didn't show up on the live site either.

Reply:
https://www.open-bio.org/board/meeting-minutes/: it's there! It's up at the top in small font, so it's not very visible. It should be lower down and bigger. Do you want to try to fix that?

Reply:
Must be a deployment delay and a caching delay on top of that.

As to the placement, in the raw HTML what I see now matches #89 (with the extra closing paragraph).

Hopefully give it another day, and my change on #90 will go live and move this below the breadcrumbs bit.